pkgs/tools/security/gnupg/20.nix

```diff
@@ -3,13 +3,13 @@
 
 # Each of the dependencies below are optional.
 # Gnupg can be built without them at the cost of reduced functionality.
-, pinentry ? null, x11Support ? true
+, pinentry ? null, guiSupport ? true
```

Comment:
@kirelagin 
By the way I've discovered that pinentry the only way to interactively enter gpg key passphrases. It's not only providing gui but also tui (simple terminal, ncurses), so ```guiSupport``` is not so accurate either. So I would revert https://github.com/NixOS/nixpkgs/commit/6b9bb7231958340d21ed15c3b5e730710a3b9d14 as it provides no advantage over ```gnupg.override { pinentry = null }``` interface.

[offtopic]Nice meeting you here. The world is small :)[/offtopic]